frontend/src/app/modules/interventions/list/table/table.component.html

```diff
@@ -2,14 +2,14 @@
   <mat-table [dataSource]="dataSource">
     <!-- Position Column -->
     <ng-container matColumnDef="position">
-      <mat-header-cell *matHeaderCellDef>Numer</mat-header-cell>
+      <mat-header-cell *matHeaderCellDef>L.p.</mat-header-cell>
```

Comment:
Changing to a shorter label on desktop view, since previous version was going beyond its container on my  screen